sbpy/units/tests/test_core.py

```diff
@@ -90,3 +90,33 @@ def test_spectral_density_vega_bp(filename, fluxd, to, tol):
         assert np.isclose(v.value, to.value, atol=tol)
     else:
         assert np.isclose(v.value, to.value, rtol=tol)
+
+
+@pytest.mark.parametrize('mag, wfb, M_sun, ref', (
+    (3.4 * VEGAmag, None, None, 0.02865984),
+    (3.4 * VEGAmag, 'johnson_v', None, 0.02865984),
+    (3.4 * VEGAmag, 'johnson_i', None, 0.01387486),
+    (3.4 * VEGAmag, 5500 * u.AA, None, 0.02774623),
+    (3.4 * VEGAmag, None, -25.0 * VEGAmag, 0.14694669),
+    (3.4 * u.ABmag, None, -25.0 * u.ABmag, 0.14694669),
+))
```

Comment:
Are there any literature values that we can test against?

Reply:
Unfortunately I don't think so.  The best would be to use some  objects, such as Ceres, for which we have good measurements for size, magnitude, and reflectance.  The test for `M_sun=-25` and for `ABmag` is just to verify the code works, but there should be a better way.  Will revise it.

Reply:
Using Ceres sounds like a great test.

Reply:
Well, that's what I've been doing with all the `3.4 * VEGAmag` :)